application/src/main/java/org/togetherjava/tjbot/commands/utils/StringDistances.java

```diff
@@ -65,6 +65,14 @@ public static <S extends CharSequence> Optional<S> autocomplete(CharSequence pre
      */
     public static Collection<String> closeMatches(CharSequence prefix,
             Collection<String> candidates, int limit) {
+        if (prefix.isEmpty()) {
+            return candidates.stream().sorted().limit(limit).toList();
+        }
+
+        if (candidates.isEmpty()) {
+            return candidates;
```

Comment:
personally, i prefer `return List.of();` in such a case. makes it more explicit and clear that the returned list is empty.